uvm/create.go

```diff
@@ -23,6 +23,7 @@ type UVMOptions struct {
 	OperatingSystem         string                  // "windows" or "linux".
 	Resources               *specs.WindowsResources // Optional resources for the utility VM. Supports Memory.limit and CPU.Count only currently. // TODO consider extending?
 	AdditionHCSDocumentJSON string                  // Optional additional JSON to merge into the HCS document prior
+	LockedMemory            bool
```

Comment:
Would this be better as an enum/string constants in case we have additional backing types in the future?